src/app/clusters/scenes/scenes.cpp

```diff
@@ -186,13 +177,43 @@ void emAfPluginScenesServerPrintInfo()
             emberAfCoreFlush();
 #endif // ZCL_USING_COLOR_CONTROL_CLUSTER_SERVER
 #ifdef ZCL_USING_DOOR_LOCK_CLUSTER_SERVER
-            emberAfCorePrint(" door %x", entry.lockStateValue);
+            if (entry.lockStateValue.IsNull())
+            {
+                emberAfCorePrint(" door null");
+            }
+            else
+            {
+                emberAfCorePrint(" door %x", to_underlying(entry.lockStateValue.Value()));
+            }
 #endif
 #ifdef ZCL_USING_WINDOW_COVERING_CLUSTER_SERVER
-            emberAfCorePrint(" Window percentage Lift %3u, Tilt %3u", entry.currentPositionLiftPercentageValue,
-                             entry.currentPositionTiltPercentageValue);
-            emberAfCorePrint(" Window percent100ths Lift %5u, Tilt %5u", entry.targetPositionLiftPercent100thsValue,
-                             entry.targetPositionTiltPercent100thsValue);
+            if (!entry.currentPositionLiftPercentageValue.IsNull() && !entry.currentPositionTiltPercentageValue.IsNull())
+            {
+                emberAfCorePrint(" Window percentage Lift %3u, Tilt %3u", entry.currentPositionLiftPercentageValue.Value(),
+                                 entry.currentPositionTiltPercentageValue.Value());
+            }
+            else if (!entry.currentPositionLiftPercentageValue.IsNull())
+            {
+                emberAfCorePrint(" Window percentage Lift %3u", entry.currentPositionLiftPercentageValue.Value());
+            }
+            else if (!entry.currentPositionTiltPercentageValue.IsNull())
+            {
+                emberAfCorePrint(" Window percentage Tilt %3u", entry.currentPositionTiltPercentageValue.Value());
+            }
+
+            if (!entry.targetPositionLiftPercent100thsValue.IsNull() && !entry.targetPositionTiltPercent100thsValue.IsNull())
+            {
+                emberAfCorePrint(" Window percent100ths Lift %5u, Tilt %5u", entry.targetPositionLiftPercent100thsValue.Value(),
+                                 entry.targetPositionTiltPercent100thsValue.Value());
+            }
+            else if (!entry.targetPositionLiftPercent100thsValue.IsNull())
+            {
+                emberAfCorePrint(" Window percent100ths Lift %5u", entry.targetPositionLiftPercent100thsValue.Value());
+            }
+            else if (!entry.targetPositionTiltPercent100thsValue.IsNull())
+            {
+                emberAfCorePrint(" Window percent100ths Tilt %5u", entry.targetPositionTiltPercent100thsValue.Value());
```

Comment:
Can't we create a kind of Percent/Percent100ths type display function later ?

Reply:
Yes, we can.  I suspect a lot of this code will change as we fix up scenes in general...